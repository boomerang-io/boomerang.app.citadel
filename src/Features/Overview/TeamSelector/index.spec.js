import React from "react";
import { TeamSelector } from "./index";

const mockfn = jest.fn();
const teams = [
  {
    id: "5a8b331e262a70306622df73",
    name: "ATT",
    higherLevelGroupId: "59aebcf47424530fce952fa2",
    ucdApplicationId: "96c22f37-5443-4bf5-83bb-7ce90deec820",
    isActive: true,
    audits: [
      {
        auditerId: null,
        date: 1519072030853,
        note: "CI-Team created"
      }
    ],
    boomerangTeamName: "AT&T MIL Mobile@Scale",
    boomerangTeamShortname: "ms-att-mil"
  }
];

describe("TeamSelector --- Snapshot", () => {
  it("Capturing Snapshot of TeamSelector", async () => {
    const { baseElement } = global.rtlRender(<TeamSelector handleChangeTeam={mockfn} teams={teams} />);
    expect(baseElement).toMatchSnapshot();
  });
});
