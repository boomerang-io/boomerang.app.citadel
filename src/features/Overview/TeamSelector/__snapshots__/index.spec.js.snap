// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header --- Snapshot Capturing Snapshot of Header 1`] = `
<div
  className="container"
>
  <h2
    className="title"
  >
    Team
  </h2>
  <div
    className="bx--bmrg-select"
  >
    <div
      className="bx--list-box__wrapper"
    >
      <div
        className="bx--combo-box bx--list-box"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="listbox"
        tabIndex="-1"
      >
        <div
          aria-controls={null}
          aria-expanded={false}
          aria-haspopup={true}
          aria-label="Open menu"
          aria-owns={null}
          className="bx--list-box__field"
          data-toggle={true}
          onBlur={[Function]}
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={-1}
          type="button"
        >
          <input
            aria-activedescendant={null}
            aria-autocomplete="list"
            aria-controls="team__menu"
            aria-expanded={false}
            aria-label="Choose an item"
            autoComplete="off"
            className="bx--text-input"
            disabled={false}
            id="team"
            onBlur={[Function]}
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="Select a team"
            role="combobox"
            title="Team"
            value=""
          />
          
          <div
            className="bx--list-box__menu-icon"
          >
            <svg
              aria-label="Open menu"
              focusable="false"
              height={16}
              name="chevron--down"
              preserveAspectRatio="xMidYMid meet"
              role="img"
              style={
                Object {
                  "willChange": "transform",
                }
              }
              viewBox="0 0 16 16"
              width={16}
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8 11L3 6l.7-.7L8 9.6l4.3-4.3.7.7z"
              />
              <title>
                Open menu
              </title>
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
